bin/generate-css-utility-classes-docs.js

```diff
@@ -43,4 +187,20 @@ const stylesheetsDirectory = path.resolve(
   } catch (error) {
     throw new Error('Error generating the CSS utilty class Storybook stories');
   }
-})();
+}
+
+if (process.env.NODE_ENV !== 'test') {
```

Comment:
When this runs before Storybook is built, the node environment is always undefined, so I'm checking for `"test"` which jest provides for a node environment.